contract/README.rst

```diff
@@ -21,9 +21,20 @@ To use this module, you need to:
 
 #. Go to Sales -> Contracts and select or create a new contract.
 #. Check *Generate recurring invoices automatically*.
-#. Fill fields and add new lines. You have the possibility to use markers in
-   the description field to show the start and end date of the invoiced period.
-#. A cron is created with daily interval, but if you are in debug mode can
+#. Fill fields for selecting the recurrency and invoice parameters:
+   * Journal
+   * Pricelist
+   * Period. It can be any interval of days, weeks, months, months last day or
+     years.
```

Comment:
Travis RST checks dislike this linebreak: https://travis-ci.org/OCA/contract/jobs/164503506#L312